recipes/disabled.rb

```diff
@@ -17,9 +17,12 @@
 # limitations under the License.
 #
 
-unless node['splunk']['disabled']
-  Chef::Log.debug('The chef-splunk::disabled recipe was added to the node,')
-  Chef::Log.debug('but the attribute to disable splunk was not set.')
+if node['splunk']['disabled'] == false
```

Comment:
same here this should be left `unless node['splunk']['disabled']`

Reply:
depends...some style guides will suggest it's not good to use `unless` this purpose by itself; however, that's not my main concern. My response to explicitly checking for a value is the same as the last comment.

Reply:
```
irb(main):001:0> x = 'true'
=> "true"
irb(main):002:0> if x
irb(main):003:1> puts 'it was true'
irb(main):004:1> else
irb(main):005:1> puts 'it is not true'
irb(main):006:1> end
it was true
```